### ISSUE TITLE
How to switch the GPU acceleration

### ISSUE THREAD

**Issue by ffli** [ue4plugins/VlcMedia#81]
In the source code i have tried the two ways to start the gpu acceleration: first is in the libvlcInstance add the config options of "--avcodec-hw=d3d11va", the second is instance the libvlc.dll libvlc_media_add_option interface, neither they are visible. The second way even the vlcMedia can not load, In the vlcModule can not load and initialize. Anyone achieves the gpu acceleraton by this plugin. thanks.

**Comment by eusru** [comment]
I have tried the first way to as same as your .  
After i enabled ,seems CPU is still working under high load and GPU is also!! 
Nothing changed........

**Comment by ffli** [comment]
Yes, so i think, if the libvlc libray or dynamic library is not open its gpu interface or others. In the vlc player, when i use the gpu acceleration the cpu working down load, but i can not debug to the libvl code, so i cannot decide what is the problem.&nbsp;Thank you for your reply.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年2月24日(星期四) 中午11:18
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ue4plugins/VlcMedia] How to switch the GPU acceleration (#81)





 
I have tried the first way to as same as your .
 After i enabled ,seems CPU is still woking under high load and gpu is also!!Nothing changed........
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by eusru** [comment]
The result is that I just tested the rtsp video stream, and I am not sure about other formats of video.
Maybe other formats of video play as you said
Finally, barely used.
Thx for your reply.
If you have any new idea about this ,please share with me!!!

**Comment by huajianbu** [comment]
Has the author solved this problem? I've tried many parameters, but there's been no response. i used like ":avcodec-hw=any""--avcodec-hw=any"":avcodec=any""--avcodec=any"
